src/react-native-animated-charts/src/charts/linear/ChartPath.js

```diff
@@ -5,7 +5,6 @@ import React, {
   useRef,
   useState,
 } from 'react';
-import { Platform } from 'react-native';
```

Comment:
It's inside the lib. Please do not change it here, because it will not be in sync with the package version

Reply:
Ooops. Global find and replace 😅